dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsAuthenticationTest.java

```diff
@@ -79,7 +79,13 @@ private void addFeature(String feature) throws Exception {
     }
 
     private void waitForMetricsEndpoint(LibertyServer server) throws Exception {
-        assertNotNull("Web application is not available at */metrics/", server.waitForStringInLog("CWWKT0016I.*/metrics/", TIMEOUT * 2, server.getDefaultLogFile()));
+        assertNotNull("Web application is not available at */metrics/", server.waitForStringInLogUsingMark("CWWKT0016I.*/metrics/", server.getDefaultLogFile()));
+    }
+
+    private void waitForPrerequisites(LibertyServer server) throws Exception {
+        assertNotNull("TCP Channel defaultHttpEndpoint has not started (port 8010)", server.waitForStringInLog("CWWKO0219I.*8010.*", server.getDefaultLogFile()));
+        assertNotNull("TCP Channel defaultHttpEndpoint-ssl has not started (port 8020)", server.waitForStringInLog("CWWKO0219I.*8020.*", server.getDefaultLogFile()));
```

Comment:
Do not hardcode port 8020.

Reply:
Fixed using System.getProperty("bvt.prop.HTTP_default.secure", "8020")